docs/tutorials/collaboration/3.sharing.md

```diff
@@ -0,0 +1,52 @@
+# Sharing
+
+## Introduction
+
+In AI DIAL, *resources* refers to conversations, prompts, files, and applications. When a resource is created or uploaded to DIAL, it is stored in a dedicated storage bucket. You have the ability to share these resources with other users or applications. This sharing functionality is available through both the DIAL API and the AI DIAL Chat UI.
+
+## DIAL API
+
+There are two sides in the sharing process: the resource owner and the resource recipient. AI DIAL API has endpoints to facilitate workflows for either of them.
+
+### Endpoints for resource owners
+
+* `/create`: resource owners can use this endpoint to share a resource by creating an `invitationLink` which includes an `invitationId`. On top of that, you can grant **read** or **write** access to your resource. **Note**: If an app/user changes the shared resource, HTTP **etag** is used by AI DIAL Core to determine versioning and resolve conflicts. You can use [Subscription API](/tutorials/collaboration/notifications#subscriptions) to get notifications about changes made by other users or applications. Refer to [DIAL API](https://epam-rail.com/dial_api#tag/Sharing/paths/~1v1~1ops~1resource~1share~1create/post) to view examples. **Limitations**: the system does not support getting all the recipients of a shared resource.
+* `/copy`: this endpoint is used when you need to modify a shared resource. For example, if you have a conversation with attachments that has been shared, and you decide to add another attachment, this endpoint ensures that other users can access the updated shared resource with the new attachment included.
```

Comment:
you need to add permissions to users for a new resource.

Reply:
fixed